File: app/src/main/java/com/tutv/android/ui/series_carrousel/tv_banner/TvBannerView.java
package com.tutv.android.ui.series_carrousel.tv_banner;

public interface TvBannerView {
    void setImageUrl(String imageUrl);

    void setTitle(String text);

    void setSubtitle(String text);

    TvBannerPresenter getPresenter();

    void navigateToSeriesPage(int seriesId);

    void onDetach();

    void setSeriesFollowed(boolean followed);

    void showError(String error);
}

File: app/src/main/java/com/tutv/android/ui/series_carrousel/tv_banner/TvBannerView.kt
package com.tutv.android.ui.series_carrousel.tv_banner

interface TvBannerView {
    fun setImageUrl(imageUrl: String?)
    fun setTitle(text: String?)
    fun setSubtitle(text: String?)
    fun getPresenter(): TvBannerPresenter?
    fun navigateToSeriesPage(seriesId: Int?)
    fun onDetach()
    fun setSeriesFollowed(followed: Boolean?)
    fun showError(error: String?)
}